packages/grid/src/useGrid.ts

```diff
@@ -0,0 +1,359 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useState, HTMLProps, HTMLAttributes, KeyboardEvent } from 'react';
+import { composeEventHandlers } from '@zendeskgarden/container-utilities';
+
+const gridKeys = ['ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown', 'Home', 'End'];
```

Comment:
Should this array be via an enum that can be reused in the switch statements (similar to [this](https://github.com/zendeskgarden/react-components/blob/main/packages/forms/src/elements/file-list/utils.tsx))? Also [nit] we tend to SNAKE_CASE constants.